src/utils/calculateResults.ts

```diff
@@ -36,10 +36,10 @@ export function calculateVotes(
           voterIds: new Set(),
         };
       }
-      projectVotes[vote.projectId]!.amounts.push(vote.amount);
+      projectVotes[vote.projectId]!.amounts.push(vote.amount ** 0.5);
       projectVotes[vote.projectId]!.voterIds.add(ballot.voterId);
-      projectVotes[vote.projectId]!.total += vote.amount;
-      actualTotalVotes += vote.amount;
+      projectVotes[vote.projectId]!.total += vote.amount**0.5;
+      actualTotalVotes += vote.amount**0.5;
```

Comment:
wow, cool that its only a tiny few tweaks to the code in practice, considering how fancy QF sounds in practice...

Reply:
yeah, ill define a constant for the 0.5, but I couldn't see the votes in results page until I commented the publishedAt filter [here](https://github.com/ObolNetwork/easy-retro-pgf/blob/main/src/server/api/routers/results.ts#L104), not sure why i have it as null in my db and couldn't find where it should be set, will check more tomorrow

<img width="1415" alt="Screenshot 2024-09-30 at 9 24 53 PM" src="https://github.com/user-attachments/assets/a4578eec-c362-4fa1-a8c8-9886c7e81792">